### DISCUSSION TITLE
New Entry button label, include entry type

### DISCUSSION THREAD

**Discussion by RichardJong** [craftcms/cms#8542] [Ideas] [answered]
### Description
When on an Entry index page, I want the add button to display 'New {section} entry' instead of 'New Entry'.
I see this is already programmed in EntryIndex.js:98, but it always shows 'New entry'

`label = (this.settings.context === 'index' ? Craft.t('app', 'New entry') : Craft.t('app', 'New {section} entry', {section: selectedSection.name}));`

**Comment by richhayler** [comment]
It might be better instead of "New {section} entry" to use "New {section.entry.type}", the "entry type" being the one at the top of the list if additional entry types are defined for the selected section?

In my experience, sections usually have plural names as they contain a list of entries, e.g. "articles, posts, clients, projects, etc". And entry types are defined as singular, e.g. "article, post, client, project", so taking this approach it would be great if the "New Entry" button was contextual and displayed as "New article, New post, New client, New project, etc".

**Comment by brandonkelly** [comment, marked as answer]
This was requested previously by #9429 so marking this as a duplicate.
